### ISSUE TITLE
MNI305 fails as output space

### ISSUE THREAD

**Issue by dkp** [templateflow/templateflow#147]
When I try to use MNI305 as an outspace for fmriprep, I get an error and the process fails.  

Here's my docker call:
```bash
docker run --rm -it \
 -v /Users/dpat/license.txt:/opt/freesurfer/license.txt:ro \
 -v "${PWD}"/inputs/bids:/data:ro \
 -v "${PWD}"/outputs/derivatives:/out \
 -v "${PWD}"/fmriprep_work:/work \
nipreps/fmriprep:22.1.1 /data /out participant --participant_label CAM003 -w /work --stop-on-first-crash --ignore slicetiming  --fs-no-reconall -v -v  --output-spaces MNI305
```

And here's the error:

```bash
	traits.trait_errors.TraitError: The 'brain_mask' trait of a _TemplateFlowSelectOutputSpec instance must be a pathlike object or string representing an existing file, but a value of '[]' <class 'str'> was specified.

	During handling of the above exception, another exception occurred:

	Traceback (most recent call last):
	  File "/opt/conda/lib/python3.9/site-packages/nipype/interfaces/base/core.py", line 401, in run
	    outputs = self.aggregate_outputs(runtime)
	  File "/opt/conda/lib/python3.9/site-packages/nipype/interfaces/base/core.py", line 461, in aggregate_outputs
	    raise FileNotFoundError(msg)
	FileNotFoundError: No such file or directory '[]' for output 'brain_mask' of a TemplateFlowSelect interface
```
Can anyone help me understand what is failing here?

Thank you,

Dianne

**Comment by mgxd** [comment]
This is likely occurring because the MNI305 templateflow files do not include the `res` entity, but fmriprep/smriprep expect this to be present when fetching the relevant files. I've opened up #149 for a broader discussion

**Comment by mgxd** [comment]
@dkp this should be fixed with the latest fmriprep series (23.2.x) - please open an issue if still encountering problems.
